src/oncall/ui/static/js/oncall.js

```diff
@@ -926,6 +939,46 @@ var oncall = {
           .append('<label class="label-col">Slack</label>')
           .append('<span class="data-col">' + userData.contacts.im + '</span>')
         )
+      },
+      setupEscalateModal: function(){
+        var $modal = $(this.data.escalateModal),
+            $modalForm = $modal.find('.modal-form'),
+            $modalInput = $modalForm.find('.create-input'),
+            $modalBtn = $modal.find('#escalate-btn'),
+            self = this;
+
+        $modal.on('shown.bs.modal', function(e){
+          $modalInput.trigger('focus');
+        });
+
+        $modalBtn.on('click', function(e){
+          e.preventDefault();
+          $modal.find('.alert').remove();
+          self.escalate($modalForm.find('#escalate-description').val());
```

Comment:
unless this call is instantaneous, you should add some loading/disabled states here to prevent multiple clicks and give user feedback that its escalating. See oncall.team.createTeam method for example.